lib/pure/uri.nim

```diff
@@ -153,6 +160,42 @@ func encodeQuery*(query: openArray[(string, string)], usePlus = true,
       result.add('=')
       result.add(encodeUrl(val, usePlus))
 
+iterator decodeQuery*(data: string): tuple[key, value: TaintedString] =
+  ## Reads and decodes query string ``data`` and yields the (name, value) pairs the
```

Comment:
optional nit: `(name, value)` => `(key, value)` ?

Reply:
Is it better to change `tuple[key, value: TaintedString]` to tuple[name, value: string] ?

Reply:
hmm, may be best to be consistent with std/cgi (and key/value is more common in stdlib, eg tables etc)